WalletWasabi.Tests/UnitTests/AllFeeEstimateTests.cs

```diff
@@ -303,7 +303,22 @@ public async Task RealWorldMempoolMinFeeAsync()
 		var estimations = feeRates.Estimations;
 		var minFee = estimations.Min(x => x.Value);
 
-		Assert.Equal(30, minFee); // this is the calculated MempoolMinFee needed to be in the top 200MB
+		Assert.Equal(20, minFee); // this is the calculated MempoolMinFee needed to be in the top 200MB
+	}
+
+	[Fact]
+	public async Task RealWorldMempoolRpcMinFeeAsync()
+	{
+		var mockRpc = CreateAndConfigureRpcClient(hasPeersInfo: true);
+		var mempoolInfo = MempoolInfoGenerator.GenerateRealBitcoinKnotsMemPoolInfo();
+		mockRpc.Setup(rpc => rpc.GetMempoolInfoAsync(It.IsAny<CancellationToken>())).ReturnsAsync(mempoolInfo);
+		mockRpc.Setup(rpc => rpc.EstimateSmartFeeAsync(It.IsAny<int>(), EstimateSmartFeeMode.Conservative, It.IsAny<CancellationToken>()))
+			.ReturnsAsync(FeeRateResponse(2, 0m)); // all estimations a 0 s/b so, only estimations based on mempool.
+		var feeRates = await mockRpc.Object.EstimateAllFeeAsync(EstimateSmartFeeMode.Conservative);
```

Comment:
Nit:
`EstimateSmartFeeMode.Conservative` is the default value of that parameter.
```suggestion
		var feeRates = await mockRpc.Object.EstimateAllFeeAsync();
```

Reply:
Yes but the mock expects that value. I can be wrong but anyway, if Kimi removes the parameter then this will be removed too.